taworks/taform/models.py

```diff
@@ -69,6 +73,32 @@ class Meta:
         widgets = {'past_position_one':Textarea(attrs={'cols':80,'rows':5}), 'past_position_two':Textarea(
             attrs={'cols':80, 'rows':5}), 'past_position_three':Textarea(attrs={'cols':80, 'rows':5})}
 
+class AssignTA(ModelForm):
+    def __init__(self, *args, **kwargs):
+        super(AssignTA,self).__init__(*args, **kwargs)
+        self.fields['term'].widget.attrs['readonly']=True
+        self.fields['term'].disabled
+        self.fields['course_subject'].widget.attrs['readonly']=True
+        self.fields['course_subject'].disabled
+        self.fields['course_id'].widget.attrs['readonly']=True
+        self.fields['course_id'].disabled
+        self.fields['section'].widget.attrs['readonly']=True
+        self.fields['section'].disabled
+        self.fields['course_name'].widget.attrs['readonly']=True
+        self.fields['course_name'].disabled
+        self.fields['instructor_name'].widget.attrs['readonly']=True
+        self.fields['instructor_name'].disabled
+        self.fields['instructor_email'].widget.attrs['readonly']=True
+        self.fields['instructor_email'].disabled
+    class Meta:
+        model = Course
+        fields = '__all__'
+        exclude = ('url_hash',)
+        widgets = {'full_ta':Textarea(attrs={'cols':3,'rows':1, 'style':'resize:none;'}),
```

Comment:
https://trello.com/c/cWrlTxjD/68-submitting-letters-in-number-of-tas-page

left a comment there could fix it here or next sprint i'm happy either way